docs/user-guide/nodes.rst

```diff
@@ -396,18 +396,20 @@ On the Pulp server to be used as the parent node:
 
 1. Install the pulp-nodes-parent package group.
 
+2. Configure and enable :ref:`OAuth <oauth-config>`.
+
```

Comment:
@jortel Thanks for pointing this out. I pushed a new version. Would you be able to take another look?